docs/index.md

```diff
@@ -18,38 +18,20 @@ To install all Arcus.WebApi packages:
 PM > Install-Package Arcus.WebApi.All
 ```
 
-To install the Arcus.WebApi.Logging package:
-
-```shell
-PM > Install-Package Arcus.WebApi.Logging
-```
-
-To install the Arcus.WebApi.OpenApi.Extensions package:
-
-```shell
-PM > Install-Package Arcus.WebApi.OpenApi.Extensions
-```
+For more granular packages we recommend reading the documentation.
 
 # Features
 
-## Arcus.WebApi.Security
-
-The `Arcus.WebApi.Security` package contains functionality to easily add security capabilities to an API.
```

Comment:
Some "small" descriptions of each sub-section could be useful?
I like the structure; but maybe still add some small descriptions on each one?

Reply:
What information would you like to be added to this view?

![image](https://user-images.githubusercontent.com/4345663/62353589-3f227000-b50b-11e9-9445-23c6f2600d5b.png)

Not saying I don't want to change it, just trying to see what is expected!

Reply:
As discussed it's good as it is today